### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by jrosario19**
### In this PR I added the following features:
-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,6 +8,41 @@ def index
...
+    redirect_to user_posts_path(current_user)
+  end
+
+  def comment
+    @post = Post.find(params[:id])
+    p params
+    @comment = Comment.new(author: current_user, post: @post, text: params[:text])
+    @comment.save
+    redirect_to user_posts_path(current_user)
+  end
```
Comment (CollinsTatang) [inline app/controllers/posts_controller.rb]: ### **Great** implementation of the project requirements.

- [ ] Kindly create a `CommentsController` action for post comments and refactor your code.

- A comment is created on behalf of the currently logged in user (current_user) of the app. The comments controller should have new and create actions.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -8,6 +8,41 @@ def index
...
+    else
+      render :new
+    end
+  end
+
+  def like
+    @post = Post.find(params[:id])
+    Like.new(author: current_user, post: @post).save
+    redirect_to user_posts_path(current_user)
+  end
```
Comment (CollinsTatang) [inline app/controllers/posts_controller.rb]: ### **Great** implementation of the project requirements.

- [ ] Kindly create a ``LikesController`` action for post likes and refactor your code.

- A user should be able to add a like to posts. The likes controller should have a create actions

- There should be a route in the config/routes.rb file for creating like.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @jrosario19, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ahmetbozaci — APPROVED** [review]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 



@jrosario19  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯 Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Well done with creating another controller for comments and likes.✔️
- I like the style of the page.✔️

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] In the schema user model has a photo column. Photos should come from the schema

image

___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**